src/main/java/de/tum/in/www1/artemis/service/FileUploadSubmissionExportService.java

```diff
@@ -30,7 +32,7 @@ protected void saveSubmissionToFile(Exercise exercise, Submission submission, Fi
 
         // we need to get the 'real' file path here, the submission only has the api url path
         String filePath = FileUploadSubmission.buildFilePath(exercise.getId(), submission.getId());
-        String[] apiFilePathParts = ((FileUploadSubmission) submission).getFilePath().split("/");
+        String[] apiFilePathParts = ((FileUploadSubmission) submission).getFilePath().split(Pattern.quote(FileSystems.getDefault().getSeparator()));
```

Comment:
What about file separator ?

Reply:
Didn't know about this one. Implemented.